src/qt/researcher/researcherwizardemailpage.h

```diff
@@ -2,6 +2,7 @@
 #define RESEARCHERWIZARDEMAILPAGE_H
 
 #include <QWizardPage>
+#include <QRegularExpressionValidator>
 
 class ResearcherModel;
```

Comment:
Minor: since `boincEmailValidator` is a pointer type, we can move the `#include` to researcherwizardemailpage.cpp and forward-declare `QRegularExpressionValidator` to avoid the header bloat.

```suggestion

class ResearcherModel;
class QRegularExpressionValidator;
```

Reply:
@a123b You want to make this change? I think the above suggestion was only part of it. Then I will approve.

Reply:
Done